cmd/iprepd/main.go

```diff
@@ -58,7 +58,7 @@ func main() {
 
 func IPBlocklistGCS(config iprepd.ServerCfg, reputationDump []iprepd.Reputation) error {
 	var (
-		blocklistFile       = "./ip-blocklist.txt"
+		blocklistFile       = "./ip-blocklist"
```

Comment:
So this is really dumb, but the firewalls would complain and error out if the extension was .txt (you can't make this stuff up). Simply removing the extension, caused the error to go away, and everything to start working :\

Reply:
This is just the local file, the path it gets uploaded to is in the config. This is fine to change but its the config change that will do what you need.